app/Directory.Packages.props

```diff
@@ -43,5 +43,8 @@
     <PackageVersion Include="System.Linq.Async" Version="6.0.1" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.5.4" />
     <PackageVersion Include="xunit" Version="2.6.2" />
+    <PackageVersion Include="ApprovalTests" Version="5.2.4" />
+    <PackageVersion Include="FluentAssertions" Version="6.8.0" />
+    <PackageVersion Include="Moq" Version="4.20.70" />
```

Comment:
I believe we should instead use NSubstitute, as there were issues last year with Moq.